README.rst

```diff
@@ -990,6 +992,106 @@ so they can be overridden if needed.
 In the example above, the "SSID" template is flagged as "(required)"
 and its checkbox is always checked and disabled.
 
+How to configure tunnels using WireGuard
+----------------------------------------
+
+Follow the procedure described below to setup WireGuard tunnels on your devices.
+
+**Note:** This example uses **Shared systemwide (no organization)** option as
+the organization for VPN server and VPN client template. You can use any
+organization as long as VPN server, VPN client template and Device has same
+organization.
+
+1. Create VPN server configuration for WireGuard
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+1. Visit ``/admin/config/vpn/add/`` to add a new VPN server.
+2. We will set **Name** of this VPN server ``Wireguard`` and **Host** as
+   ``wireguard-server.mydomain.com`` (update this to point to your
+   WireGuard VPN server).
+3. Select ``WireGuard`` from the dropdown as **VPN Backend**.
+4. OpenWISP does IP address management for *WireGuard VPN backend*. You can
+   create a new subnet or select an existing one from the dropdown menu. You
+   can also assign an **Internal IP** to the WireGuard Server or leave it empty
+   for OpenWISP to configure.
```

Comment:
I'd end with something like:

leave it empty and OpenWISP will create one automatically using the first available IP address in the subnet.